.env.example

```
@@ -1,18 +1,28 @@
+APP_POSTGRES_PASSWORD=change-me-for-shared-environments
+
+# Local business PostgreSQL source startup
+BUSINESS_POSTGRES_SOURCE_DB=business
+BUSINESS_POSTGRES_SOURCE_USER=source_reader
+BUSINESS_POSTGRES_SOURCE_PASSWORD=change-me-for-local-source-topology
+
+# Local business MSSQL source startup
+BUSINESS_MSSQL_SOURCE_SA_PASSWORD=ChangeMeDevOnly_123
 
 # Backend application settings
 SAFEQUERY_APP_NAME=SafeQuery API
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)